### PR TITLE
update status on plugin CSI migration

### PR DESCRIPTION
**Pull request by louisportay**
also removed two dead links

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by louisportay** [comment]
hello ?

File: sig-storage/volume-plugin-faq.md [hunk 1]
```diff
@@ -47,8 +47,7 @@ One of the goals of SIG Storage is to eventually have a CSI-compatible plugin fo
 For more details on that effort see:
 
 *   https://github.com/kubernetes/enhancements/issues/625
-*   https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-migration.md
-*   https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190129-csi-migration.md
+*   https://kubernetes.io/blog/2021/12/10/storage-in-tree-to-csi-migration-status-update/
```
Comment (xing-yang) [inline sig-storage/volume-plugin-faq.md:50]: Please add this KEP link: https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration

**Pull request merged**